app/helpers/view_helper.rb

```diff
@@ -36,7 +43,14 @@ def bat_contact_mail_to(name = 'becomingateacher<wbr>@digital.education.gov.uk',
     mail_to('becomingateacher@digital.education.gov.uk', name.html_safe, html_options)
   end
 
-  def govuk_button_link_to(body, url, html_options = {}, &_block)
+  def govuk_button_link_to(body = nil, url = nil, html_options = nil, &block)
+    if block_given?
+      body = block
```

Comment:
I think we will have to move setting the `block` till the end since setting it in the beginning is setting `url` and `html_options` to the `block` as well, also might need to switch the order to avoid `html_options` being set to the body as so: 
```
if block_given?
  html_options = url
  url = body
  body = block
end
```

Reply:
thanks @crrw, good spot, this was a last minute change because Rubocop doesn't like parallel assignment and I was just wondering why all the tests had broken.